examples/scripts/DFN_ambient_temperature.py

```diff
@@ -0,0 +1,52 @@
+#
+# Example showing how to load and solve the DFN
```

Comment:
the title could be updated to explain this is showing the ambient temperature feature

Reply:
I'll merge this PR and make that change before merging #868